geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,353 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<% set_title("Introduction to", product_name, "Clients") %>
+
+This section provides basic starting points for a variety of <%=vars.product_name %> clients, along
+with very rudimentary connect, put, get, and then a reference to more in-depth docs and examples on
+how to use the client.
+
+## Installing Apache Geode
+
+You can download Apache Geode from the website, run a Docker image, or install with homebrew on OSX.
+
+## Start an Apache Geode Cluster
+
+With a path that contains the bin directory of the installation, start a locator and server.
+
+```
+$ gfsh 
+gfsh> start locator
+gfsh> start server
+```
+
+Create a region
+
+```
+gfsh> create region --name=helloWorld --type=PARTITION 
+```
+
+Shutting Down Apache Geode
```

Comment:
"Once you are done, you can shutdown the cluster executing:"

Reply:
Done, slightly re-worded